package.json

```diff
@@ -37,9 +37,9 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && tap --coverage --reporter=spec --timeout=150 test/*.js",
-    "test-win": "tap --reporter=spec --timeout=150 test/*.js",
-    "coverage": "tap --coverage-report=lcov"
+    "pretest": "rm -rf ./node_modules/.cache",
```

Comment:
@jamestalmage Re `trash-cli`, that's a bug https://github.com/sindresorhus/trash/pull/43, but no point in having the overhead of moving it to the trash, not something you'd ever care about restoring.